samples/app/src/androidTest/java/com/sourcepoint/app/v6/TestData.kt

```diff
@@ -311,4 +282,171 @@ object TestData {
       }
     """.trimIndent()
 
+    val storedConsentV742 = """
```

Comment:
I think that would be better if we update `storedConsentV741` in order to see if consent of v7.4.2 is compatible with our tests

Reply:
v7.4.1 cannot be changed anymore let’s keep it like this